src/common.c

```diff
@@ -305,3 +306,43 @@ void raftNodeIdToString(char *output, const char *dbid, raft_node_id_t raft_id)
 {
     snprintf(output, RAFT_SHARDGROUP_NODEID_LEN + 1, "%.32s%08x", dbid, raft_id);
 }
+
+/*
+ * returns the deny_oom status for a particular set of commands
+ */
+bool isDenyOomStatus(RedisRaftCtx *rr, RaftRedisCommandArray *cmds)
```

Comment:
I don't think we need this function. We already fetch aggregated flags just before we call this function. So, we can just do "if check" there. Also, this way we won't do dictionary lookup twice. (Triggered by `CommandSpecGet()`. )

Reply:
its not needed now, but its needed for eval (or makes the code cleaner with eval).  We could pass the cs in into it.

Reply:
will try to figure out how to put the isDenyOom into CommandSpecGetAggregateFlags which should solve all issues